src/published-data/published-data.service.ts

```diff
@@ -44,8 +45,17 @@ export class PublishedDataService {
     private readonly httpService: HttpService,
     @Inject(REQUEST)
     private request: Request,
+    private configService: ConfigService,
   ) {}
 
+  async getConfig(): Promise<Record<string, unknown> | null> {
+    const config =
+      this.configService.get<Record<string, unknown>>("publishedDataConfig") ||
```

Comment:
I might missed something here, I couldn't find anywhere the file being read. 
It seems to me the config variable will be assigned with just a string of `publishedDataConfig.json` instead of  actual object?

Reply:
I think I wrote that function and completely forgot about it